packages/react/src/components/auth/SignedInOrRedirect.tsx

```diff
@@ -21,7 +20,7 @@ export const SignedInOrRedirect = (props: { children: ReactNode }) => {
     }
```

Comment:
Should the above also check for `user`? 
`if (auth && !redirected && (!isSignedIn || !user))`